iml-agent/src/env.rs

```diff
@@ -23,6 +23,10 @@ pub fn get_var(name: &str) -> String {
     env::var(name).unwrap_or_else(|_| panic!("{} environment variable is required.", name))
 }
 
+pub fn get_var_else(name: &str, default: &str) -> String {
```

Comment:
Instead of this fn, what about a `get_report_dir` fn in this file that does this internally?

```rust
get_report_dir() -> String {
   get_var_else("REPORT_DIR", "/tmp")
}
```